INSTRUCTIONS.md

```diff
@@ -100,10 +104,13 @@ Now that your computer is set up to recognize the Muse, we will configure MuLES
 
 ### A.5. Download the code for the workshop
 
-The code for the workshop consists of Python (and MATLAB) scripts that you can find [here](https://github.com/bcimontreal/bci_workshop).
+The code for the workshop consists of Python (and MATLAB / Octave) scripts that you can find [here](https://github.com/NeuroTechX/bci-workshop).
 You can download everything as a ```.zip``` file using the button ![downloadzip](fig/download_zip.jpg?raw=true "Download zip button") on the right. You then need to unzip the folder on your computer.
 
-Alternatively, if you have ```git``` installed on your computer, you can clone the repository by calling ```git clone git://github.com/bcimontreal/bci_workshop.git``` in the command line.
+Alternatively, if you have ```git``` installed on your computer, you can clone the repository by calling ```git clone https://github.com/NeuroTechX/bci-workshop.git``` in the command line.
+
+Exercise 02 uses support vector machine (SVM) algorithm to perform classification. Pre-build binaries of the [LIBSVM](http://www.csie.ntu.edu.tw/~cjlin/libsvm/) are provided for 64-bit versions of MATLAB and Octave.
```

Comment:
Tiny changes I suggest: "Exercise 02 uses the Support Vector Machine (SVM) algorithm to perform classification. Pre-built binaries of [LIBSVM](http://www.csie.ntu.edu.tw/~cjlin/libsvm/) are provided for 64-bit versions of MATLAB and Octave."